pages/10.contacts/01.manage-contacts/02.contact-monitoring/docs.en.md

```diff
@@ -288,21 +289,29 @@ Tracking codes support also [Google Analytics USERID][Google Analytics USERID] a
 
 There is a campaign action which allows you to send a custom event to [Google Analytics][Google Analytics] or [Facebook Pixel][Facebook Pixel] - it depends on there being a 'Visits a page' decision immediately before it in the campaign workflow.
 
-##### How to test Google Analytics tracking code and campaign action
+##### How to test Google Analytics tracking code and Campaign action
 
-- Install [Tag Assistant][Tag assistant] and enable recording on your website 
+- Install [Tag Assistant][Tag assistant] and enable recording on your website
 - Create campaign with the 'Visits a page' decision and 'Send tracking event' action 
 - Test it and check in the Tag Assistant debug window that you see one Pageview request and one event
 
-![Google Tag Assistant](google-analytics-tag-assistent.png)
+![Google Tag Assistant][Google Tag Assistant]
```

Comment:
This second part needs to be the filename of the image otherwise it will not work.

```suggestion
![Google Tag Assistant][google-analytics-tag-assistent.png]
```